test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/testcase/public/Common/TSRotation.cs

```diff
@@ -0,0 +1,115 @@
+        {
+            tlog.Info(tag, "Destroy() is called!");
+        }
+
+        [Test]
+        [Category("P1")]
+        [Description("Rotation constructor. With Euler Angles")]
+        [Property("SPEC", "Tizen.NUI.Rotation.Rotation C")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "CONSTR")]
+        [Property("AUTHOR", "eunkiki.hong@samsung.com")]
+        public void RotationConstructorWithEulerAngle()
+        {
+            tlog.Debug(tag, $"RotationConstructorWithEulerAngle START");
+
+            var testingTarget = new Rotation(new Radian(new Degree(30.0f)), new Radian(new Degree(45.0f)), new Radian(new Degree(60.0f)));
+            Assert.IsNotNull(testingTarget, "Can't create success object Rotation.");
+            Assert.IsInstanceOf<Rotation>(testingTarget, "Should return Rotation instance.");
+
+            testingTarget.Dispose();
+            tlog.Debug(tag, $"RotationConstructorWithEulerAngle END (OK)");
+        }
+
+        [Test]
+        [Category("P1")]
+        [Description("Rotation SetEulerAngle.")]
+        [Property("SPEC", "Tizen.NUI.Rotation.SetEulerAngle M")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "MR")]
+        [Property("AUTHOR", "eunkiki.hong@samsung.com")]
+        public void RotationSetEulerAngle()
+        {
+            tlog.Debug(tag, $"RotationSetEulerAngle START");
+
+            var testingTarget = new Rotation();
+            Assert.IsNotNull(testingTarget, "Can't create success object Rotation.");
+            Assert.IsInstanceOf<Rotation>(testingTarget, "Should return Rotation instance.");
+
+            try
+            {
+                testingTarget.SetEulerAngle(new Radian(new Degree(30.0f)), new Radian(new Degree(45.0f)), new Radian(new Degree(60.0f)));
+            }
+            catch (Exception e)
+            {
+                tlog.Debug(tag, e.Message.ToString());
+                Assert.Fail("Caught Exception : Failed!");
+            }
+
+            testingTarget.Dispose();
+            tlog.Debug(tag, $"RotationSetEulerAngle END (OK)");
+        }
+
+        [Test]
+        [Category("P1")]
+        [Description("Rotation GetEulerAngle.")]
+        [Property("SPEC", "Tizen.NUI.Rotation.GetEulerAngle M")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "MR")]
+        [Property("AUTHOR", "eunkiki.hong@samsung.com")]
+        public void RotationGetEulerAngle()
+        {
+            tlog.Debug(tag, $"RotationGetEulerAngle START");
+
+            var expectPitchRadian = new Radian(new Degree(30.0f));
+            var expectYawRadian = new Radian(new Degree(45.0f));
+            var expectRollRadian = new Radian(new Degree(60.0f));
+            var testingTarget = new Rotation(expectPitchRadian, expectYawRadian, expectRollRadian);
+            Assert.IsNotNull(testingTarget, "Can't create success object Rotation.");
+            Assert.IsInstanceOf<Rotation>(testingTarget, "Should return Rotation instance.");
```

Comment:
May I request test with negative angle?
for example

``` c#
            var expectPitchRadian = new Radian(new Degree(30.0f));
            var expectYawRadian = new Radian(new Degree(-45.0f));
            var expectRollRadian = new Radian(new Degree(60.0f));
            var testingTarget = new Rotation(expectPitchRadian, expectYawRadian, expectRollRadian);
            Assert.IsNotNull(testingTarget, "Can't create success object Rotation.");
            Assert.IsInstanceOf<Rotation>(testingTarget, "Should return Rotation instance.");
```

Reply:
Negative angle also works well. The TC what you suggested also passed.